no-computer.md

```diff
@@ -11,34 +11,43 @@ _What need is met by your idea? Who are you helping? What is that person's
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Describing games and the rules to play them does not require a lot of different terminology for many board games. Thus, a limited-scope DSL could fit this purpose. The DSL would standardize how game rules are described and enourage clarity so that users can understand them better.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+What would be really really cool is if the rules are specific enough that you could automatically generate a computer version of the game and/or an AI for the game. This idea was inspired by my idea to write up a board game (like Monopoly) for the no computer project and by a [previous final project](https://github.com/AriHC/RuleBook/blob/master/documents/final.md) from DSLs.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Standardized game rule description language.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would simply consume the "programs" in the language to learn about the game described. Readability will be key as this would technically be aimed at people, but it would be acceptable to make it more strict so that computers can parse it as well.
+
+Programming would involve taking a game (existing or theoretical) and expessing all the rules of the game (as well as some metadata) in the DSL. There isn't currently a "right way" to interact with the domain, but many games have rule books that are at least somewhat similar to each other. Looking at these could be a good starting point.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+It's not entirely clear what a "program" in this language should do, but it should be clear what someone can do after reading in a "program." People should be able to play the game in question as well as perhaps detect logic errors in the game's rules description. They should even be able to act as a host for the game and make sure all players are following the rules.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+Ideally, it would be easy to describe the rules for major board games. It may also be possible to describe rules for different kinds of games (like card games), although it should be clear that it will be hard to make a DSL that can describe _any_ game's rules well. There will be some games that are just way to different from board games that will not work well with this DSL. Obviously, it should be difficult to describe stuff that is not game rules in this language.
```

Comment:
I am curious how strange of board games could be described using this - there are a lot of board games with a lot of different kinds of constructs in their rules. I'd imagine a good number can be, but it would require a lot of creativity to make a language flexible enough to describe most board games.